backend/app/services/context/context_service.py

```
@@ -1536,3 +1645,92 @@ def get_unlinked_contexts(
+            )
+            context.status = ContextStatus.FAILED.value
+            context.error_message = str(e)
+            context.extracted_text = ""
+            context.text_length = 0
+
+        except Exception as e:
+            logger.exception(
+                f"Unexpected error parsing document for context {context_id}: {e}"
+            )
+            context.status = ContextStatus.FAILED.value
+            context.error_message = f"Parse failed: {str(e)}"
```

Ruff (0.15.9)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)